packages/core/src/electron-browser/window/electron-window-service.ts

```diff
@@ -14,20 +14,19 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { injectable } from 'inversify';
-import { ipcRenderer } from 'electron';
+import { injectable, inject } from 'inversify';
 import { NewWindowOptions } from '../../browser/window/window-service';
 import { DefaultWindowService } from '../../browser/window/default-window-service';
+import { ElectronWindowService as ElectronMainWindowService } from '../../electron-common/electron-window-service';
```

Comment:
there are r files with `electron-window-service` name? https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#unique-names

Reply:
Good point, I have also noticed it. I will pick another name. Thanks!

Reply:
btw `ElectronMainWindowService` sounds good